modepy/quadrature/witherden_vincent_quad_data.py

```diff
@@ -0,0 +1,2106 @@
+# GENERATED by modepy/contrib/weights-from-witherden-vincent.py
```

Comment:
Maybe generate a copyright/license statement here?

Reply:
For the original authors? How would that look like?

Reply:
```py
__copyright__ = "(C) Witherden and company"
__license__ = """
The glorious text of the applicable CC license
"""
```
?

Reply:
How does 73d718f look?  There doesn't seem to be a standard header for CC licenses.